source/_components/image_processing.facebox.markdown

```diff
@@ -58,3 +65,29 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+## {% linkable_title Automations %}
+
+Use the `image_processing.detect_face` events to trigger automations, and breakout the `trigger.event.data` using a [data_template](https://www.home-assistant.io/docs/automation/templating/). The following example automation sends a notification when Ringo Star is recognised:
+
+{% raw %}
+```yaml
```

Comment:
Please adjust the sequence in the YAML. I know it came from the automation editor, but that is not optimal.
```yaml
  trigger:
    - platform: event
      event_type: image_processing.detect_face
      event_data:
        name: 'Ringo_Starr'
```

Reply:
ok